rest/data/project.go

```diff
@@ -389,3 +389,54 @@ func (pc *DBProjectConnector) GetProjectFromFile(ctx context.Context, pRef model
 	}
 	return model.GetProjectFromFile(ctx, opts)
 }
+
+func DeleteBranch(ctx context.Context, pRef *model.ProjectRef) error {
```

Comment:
need to put a function comment on here -- I'd also prefer we call this "HideBranch" since we aren't actually deleting it

Reply:
Also are we using context anywhere?